.github/workflows/main.yml

```diff
@@ -2,18 +2,18 @@
 name: Main
 
 on:
-  pull_request:
+#   pull_request:
   push:
     branches:
       - 'main'
-      - 'release/*'
-      - 'force-docker-images'
-      - 'private_release/*'
-    tags:
-      - 'v[0-9]+.[0-9]+.[0-9]+*'
-  schedule:
-    # Nightly Release @ 3AM after each work day
-    - cron: "0 3 * * 2-6"
+#       - 'release/*'
+#       - 'force-docker-images'
+#       - 'private_release/*'
+#     tags:
+#       - 'v[0-9]+.[0-9]+.[0-9]+*'
+#   schedule:
+#     # Nightly Release @ 3AM after each work day
+#     - cron: "0 3 * * 2-6"
```

Comment:
I'm trying to understand this: so you are disabling main, as from now on, every workflow should be run independently of main, but then I have two questions:
- This workflow should be removed completely, right? Otherwise, it's using the old way (running those workflows via workflow_dispatch call), so it won't work on `main`
- I'm not sure the behavior we had here will be preserved. For example, the docker build wasn't running on PRs that didn't change docker files, but it will with the new workflow.

Reply:
- Yes you are right, it will be removed in next commits (when I'll ported also release workflow)
- Indeed that may not totally preserve the previous behavior, let see case by case if it's what we want (see my reply in comments for docker)